client-app/core/api/graphql/fragments/allLanguageFields.graphql

```diff
@@ -0,0 +1,9 @@
+fragment allLanguageFields on LanguageType {
+  isInvariant
+  cultureName
+  nativeName
+  threeLetterLanguageName
+  twoLetterLanguageName
+  twoLetterRegionName
+  threeLetterRegionName
```

Comment:
We don't use three letter values

Reply:
keep for compatibility with old /context method for now